hw3/hw3-v1tal4ik/src/v1tal4ik.js

```diff
@@ -0,0 +1,163 @@
+// eslint-disable-next-line max-classes-per-file
+import validateObj from './helperFunc';
+import * as objUsers from './people.data';
+
+// 1. write a constructor function `PersonFunc` whith methods : getFullName() getAge() getFullAddress();
+const PersonFunc = function (obj) {
+  const { isObjHasAllProperty, isObjPropertyTypeRight, msg } = validateObj(obj);
+  if (!isObjHasAllProperty || !isObjPropertyTypeRight) {
+    return { msg };
+  }
+  this.person = obj;
+};
+
+
+PersonFunc.prototype.getFullName = function () { return `${this.person.firstName} ${this.person.lastName}`; };
+PersonFunc.prototype.getFullAddress = function () {
+  const { country, city, street, house, apartment } = this.person.address;
+  return `${country}, ${city}, ${street}, ${house}/${apartment}`;
```

Comment:
this one still gonna throw error if apartment field is absent. It shouldn't.
Read the task description, this method should return two types of string: 
- "Ukraine, Lviv, Doroshenka street, 5/8"
- "Ukraine, Lviv, Muchna street, 7".

Reply:
nowif there is no apartment field I will get `Ukraine, Lviv, Muchna street, 7/undefined` instead of line `Ukraine, Lviv, Muchna street, 7`